.jest/setEnvVars.js

```diff
@@ -1 +1 @@
-process.env.VUE_APP_SUBCLASS_PROPERTY = 'P279';
+process.env.VUE_APP_SUBCLASS_PROPERTY_MAP = '{"default": "P279", "P171": "P171", "P131":"P131"}';
```

Comment:
I must disagree on this one.. :sweat_smile: 

Environment variables are set on this file so you can later invoke them like this:

process.env.VUE_APP_SUBCLASS_PROPERTY; as it was done in https://github.com/wmde/query-builder/pull/213/files#diff-33a89579bad0cce26e49f525b9e3d5320d953ef16e2f40f1dca270c5c0917fb4R265

If you write `process.env.VUE_APP_SUBCLASS_PROPERTY_MAP = '{"P666":"P66"}';` declaring it here is not needed. In this case you need to delete this file and also remove this line `setupFiles: [ '<rootDir>/.jest/setEnvVars.js' ],` from the file: `/.jest.config.js`

I personally prefer having them configured in the file so it behaves more like production and we have all the enviroment variables using in the unit tests in one place.

In any case this file with the current usage is not being used to it generates confusion. It should either be removed or used properly. :)

Reply:
The test being changed is a unit test and those by definition should not depend on the global state. I would rather keep this file just to make sure tests work for example if the code checks if this env variable exists and fails to do anything if not. This would make sure it runs.

Reply:
Try removing it first. It works locally but I remember having some issues once pushed... if it is not being used it should be deleted.

Reply:
I tried it out and these files are indeed no longer needed. I took the liberty to remove them by amending this PR as it is prone to cause conflicts with other work on SPARQL generation. @guergana could you have another look?

Reply:
Sure. Looks good. Will approve

Reply:
Thank you :pray: